web_widget_image_webcam/static/src/js/webcam_widget.js

```diff
@@ -0,0 +1,127 @@
+                            :: Important Note about Chrome 47+ :: < https://github.com/jhuckaby/webcamjs#important-note-for-chrome-47 >
+                            Setting "force_flash" to "true" will always run in Adobe Flash fallback mode on Chrome, but it is not desirable.
+                        */
+                        force_flash: true,
+                    });
+                }
+            });
+
+            self.$el.find('.o_form_binary_file_web_cam').off().on('click', function(){
+                // Init Webcam
+                var dialog = new Dialog(self, {
+                    size: 'large',
+                    dialogClass: 'o_act_window',
+                    title: _t("WebCam Booth"),
+                    $content: WebCamDialog,
+                    buttons: [
+                        {
+                            text: _t("Take Snapshot"), classes: 'btn-primary take_snap_btn',
+                            click: function () {
+                                Webcam.snap( function(data) {
+                                    img_data = data;
+                                    // Display Snap besides Live WebCam Preview
+                                    WebCamDialog.find("#webcam_result").html('<img src="'+img_data+'"/>');
+                                });
+                                if (Webcam.live) {
+                                    // Remove "disabled" attr from "Save & Close" button
+                                    $('.save_close_btn').removeAttr('disabled');
+                                }
+                            }
+                        },
+                        {
+                            text: _t("Save & Close"), classes: 'btn-primary save_close_btn', close: true,
+                            click: function () {
+                                var img_data_base64 = img_data.split(',')[1];
+
+                                /*
+                                    Size in base64 is approx 33% overhead the original data size.
+
+                                    Source: -> http://stackoverflow.com/questions/11402329/base64-encoded-image-size
+                                            -> http://stackoverflow.com/questions/6793575/estimating-the-size-of-binary-data-encoded-as-a-b64-string-in-python
+
+                                            -> https://en.wikipedia.org/wiki/Base64
+                                            [ The ratio of output bytes to input bytes is 4:3 (33% overhead).
+                                            Specifically, given an input of n bytes, the output will be "4[n/3]" bytes long in base64,
+                                            including padding characters. ]
+                                */
+
+                                // From the above info, we doing the opposite stuff to find the approx size of Image in bytes.
+                                var approx_img_size = 3 * (img_data_base64.length / 4)  // like... "3[n/4]"
```

Comment:
Need remove the length of the padding
```suggestion
                                var approx_img_size = 3 * (img_data_base64.length / 4) - (img_data_base64.match(/=+$/g) || []).length;  // like... "3[n/4]"
```